### DISCUSSION TITLE
Not able to make Changes in "create new order" form

### DISCUSSION THREAD

**Discussion by amanMosphe** [fleetbase/fleetbase#172] [General]
I want to change something in "Create new order" form but i am not able to do that. I am changing things and then running "docker-compose build" and then "docker-compose up -d", so the changes are going into the docker container but not reflecting in the browser. Can you please help me with this?

**Comment by shivthakker** [comment]
@amanMosphe the new order form can be changed in the order config settings - please see 'Table' - 'Config'
